src/pages/Account/Login.tsx

```diff
@@ -92,30 +92,46 @@ export const Login: React.FC<LoginProps> = ({
       <div className="hero-logo github-plain" />
       <div className="auth-form-container">
         {hasAlternativeLogins && (
-          <div className="Login__login-box">
-            <span>Quick log in with Okta</span>
-
-            <div className="Login__alternative-logins">
-              {hasCustomAuthConnect && (
-                <img
-                  className="Login__quick-login-icon"
-                  src={SAMLLoginIcon}
+          <>
+            {hasCustomAuthConnect && (
+              <div
+                className="Login__login-box"
+                onClick={signInWithSAML}
+                title="SAML SSO login"
+              >
+                <span>Quick log in with Okta</span>
+                <div
+                  className="Login__alternative-logins"
                   onClick={signInWithCustomAuth}
                   title={customAuthName}
-                  alt={customAuthName}
-                />
-              )}
-              {hasSamlAuthProviderId && (
-                <FontAwesomeIcon
-                  className="Login__quick-login-icon"
-                  icon={faGithub}
-                  size="4x"
-                  onClick={signInWithSAML}
-                  title="SAML SSO login"
-                />
-              )}
-            </div>
-          </div>
+                >
+                  <img
+                    className="Login__quick-login-icon"
+                    src={SAMLLoginIcon}
+                    alt={customAuthName}
+                  />
+                </div>
+              </div>
+            )}
+
+            {hasSamlAuthProviderId && (
+              <div
+                className="Login__login-box"
+                onClick={signInWithSAML}
+                title="SAML SSO login"
+              >
+                <span>Quick log in with Okta</span>
+                <div className="Login__alternative-logins">
+                  <FontAwesomeIcon
+                    className="Login__quick-login-icon"
+                    icon={faGithub}
+                    size="4x"
+                    title="SAML SSO login"
+                  />
+                </div>
+              </div>
+            )}
+          </>
```

Comment:
I'm not sure what the intent of this change is.. but looking at the diff here.. seems.. wrong?

Reply:
The `hasCustomAuthConnect`section  has nothing to do with Okta/SAML login.. so none of those changes should have been made here.

Reply:
Not sure why the 'quick login with' was moved/duplicated either. This should presumably remain in the `hasAlternativeLogins` block.

Reply:
The onClick handler was not applied to the whole containing div previously, so effectively only the icon itself was clickable

Reply:
Just a thought -- and this should not be a blocker -- but if we have variables like {customAuthName}  then we shouldn't have to hard-code the word _Okta_ in here, right? That would make this code more usable for other login flows...

Reply:
The intent of this change was to make the whole purple box div clickable rather than just the octocat icon. Since there's a different onclick function depending on the conditions, I had to duplicate the whole div with the "quick login with" etc.

Reply:
For some reason the diff looks weird? don't know what's going on there, but I basically deleted the existing div and replaced it with the two conditional statements

Reply:
@whoaitsaimz and I have decided to remove this change for now and move on with the other things

Reply:
> if we have variables like {customAuthName} then we shouldn't have to hard-code the word Okta in here, right? That would make this code more usable for other login flows...

@whoaitsaimz That's the plan, but the okta SSO stuff doesn't actually use the `customAuth` stuff yet (which was my more generic solution written for OHBM's SSO, but GitHub's SSO wasn't 'rolled into it')